mk/PcapPlusPlus.mk.mingw-w64

```diff
@@ -4,7 +4,7 @@
 PCAPPP_INCLUDES += -I$(PCAP_SDK_HOME)/Include
 
 # libs dir
-PCAPPP_LIBS_DIR += -L$(PCAP_SDK_HOME)/lib -L$(MINGW_HOME)/lib
+PCAPPP_LIBS_DIR += -L$(PCAP_SDK_HOME)/lib -L$(PCAP_SDK_HOME)/lib/x64 -L$(MINGW_HOME)/lib

```

Comment:
it might be a little bit confusing but MinGW-w64 doesn't mean "MinGW in 64-bit". It's actually a different fork of MinGW and supports both 32-bit and 64-bit. But because WinPcap doesn't support 64-bit, only 32-bit is actually supported

Reply:
This will make the linker found the correct library in mingw-w64 64bits mode, but I agree that only Npcap will be supported in mingw-w64_x86_64

Reply:
I'm not sure how it could work... how does the linker know which library to choose from?

Reply:
Yes linker first try to link with a 32bits library and return a warning because arch doesn't match and they try to link with the 64 one

Reply:
D:/a/_temp/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/11.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: skipping incompatible C:/Npcap-sdk/lib\wpcap.lib when searching for -lwpcap

Reply:
yes, I just saw it in the CI. Looks good to me!